pennylane/templates/embeddings/angle.py

```diff
@@ -18,10 +18,6 @@
 import pennylane as qml
 from pennylane.templates.decorator import template
 from pennylane.templates import broadcast
-from pennylane.templates.utils import (
-    check_shape,
-    get_shape,
-)
```

Comment:
Ignore of you did, but we can now delete all these utils functions!

Reply:
(Can also do this in a later PR though)

Reply:
Nice!